admin/src/components/Title.js

```diff
@@ -16,18 +16,19 @@ import PreviewIcon from "@mui/icons-material/Preview";
 import { PresetsTableCell, ValueEditorPropTypes } from "./PresetsTableCell";
 import PropTypes from "prop-types";
 import { activeRowColor } from "../styles";
-import { onChangeFieldHandler } from "./PresetsTableRow";
+import { usePresetTableRowDataState } from "./PresetsTableRow";
 import { PresetsManager } from "./PresetsManager";
 import { usePresetWidgetService } from "../services/presetWidget";
 import { useTitleWidgetService } from "../services/titleWidget";
+import { useDebounce } from "../utils";
 
-const PreviewSVGDialog = ({ open, handleClose, id }) => {
+const PreviewSVGDialog = (props) => {
```

Comment:
Id should not be passed to Dialog, might result in warning in the console
Instead, dereference props like this:
```javascript
{id, ...props}
```